README.rst

```diff
@@ -390,32 +392,32 @@ For example:
 Utils
 =====
 
-make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)
+``make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)``
```

Comment:
can you either revert this change, or remove the `\` that escapes the underscore.
Right now the entry looks like this:
```
make\_grid(tensor, nrow=8, padding=2, normalize=False, range=None, scale\_each=False, pad\_value=0)
```